runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/skills/CookingAction.java

```diff
@@ -160,4 +161,16 @@
 	private final int itemId;
 	private final int level;
 	private final float xp;
+	private final boolean isMembersOverride;
+
+	CookingAction(int itemId, int level, float xp)
+	{
+		this(itemId, level, xp, false);
+	}
+
+	@Override
+	public boolean isMembers(final ItemManager itemManager)
+	{
+		return isMembersOverride() || itemManager.getItemComposition(getItemId()).isMembers();
```

Comment:
I've realized now that the right side of these overrides should be calling `ItemSkillAction.super.isMembers(itemManager)` instead of duplicating the code it contains.

Reply:
I tried to do that originally, but couldn't figure it out. Syntax feels kinda weird, tbh